### ISSUE TITLE
Investigate use of colgroup

### ISSUE THREAD

**Issue by flack** [openpsa/grid.js#59]
Currently, if you set e.g. `hidden: true` in `colModel`, `style="display: none"` is added to each cell in the column. Performance might be better if we define a `colgroup` instead, because then the runtime for setting these properties would be O(1) instead of O(n). Markup would then look like this:

```html
<table id="grid">
    <colgroup>
        <col style="display: none">
        <col style="text-align: right">
    </colgroup>
    <tr>
        <td>This is the hidden column</td>
        <td>This is the right-aligned column</td>
    </tr>
</table>
```

**Comment by OlegKi** [comment]
I agree with that the usage of `colgroup` can be helpful in general. I consider the feature some years before. Do you tested the code? Look at [the demo](http://jsfiddle.net/OlegKi/nfjt44cr/).

The problem is that [<colgroup>](http://www.w3schools.com/tags/tag_colgroup.asp) works not exactly how one want. One can apply only some small subset of **specific** CSS properties on **visible** columns. So the real help of  `colgroup` is not so large.

**Comment by meh-uk** [comment]
It looks like you can style columns visibility - http://www.search-this.com/2007/04/11/styling-table-columns-with-css/

**Comment by flack** [comment]
Yes, I just tried that with Oleg's jsfiddle: If you replace `display: none` with `visibility: collapse`, it works as expected

**Comment by smartcorestudio** [comment]
Have you tried "visibility: collapse" on Blink-based browsers?

**Comment by smartcorestudio** [comment]
I've tested Oleg's fiddle with "visibility: collapse" on the latest Chrome and had no luck. Everything is OK on FF and IE.

**Comment by flack** [comment]
Hm, I guess you're right: Chrome (I tried 37) doesn't support that correctly

**Comment by flack** [comment]
Another option for improving rendering performance might be React:

http://facebook.github.io/react/

Unfortunately, their core lib is 130Kb, almost as big as the entire jqgrid. But maybe we can borrow a few ideas from them
